examples/connector/python/main.py

```diff
@@ -63,86 +92,85 @@ def Update(self, request, context):
 
         # -- Send UPSERT records
         for t in range(0, 3):
-            operation = connector_sdk_pb2.Operation()
+            operation = connector_sdk_pb2.UpdateResponse()
             val1 = common_pb2.ValueType()
             val1.string = "a-" + str(t)
 
             val2 = common_pb2.ValueType()
             val2.double = t * 0.234
 
             record = connector_sdk_pb2.Record()
-            record.type = common_pb2.OpType.UPSERT
+            record.type = common_pb2.UPSERT
             record.table_name = "table1"
             record.data["a1"].CopyFrom(val1)
             record.data["a2"].CopyFrom(val2)
             state["cursor"] += 1
 
             operation.record.CopyFrom(record)
-            yield connector_sdk_pb2.UpdateResponse(operation=operation)
+            yield connector_sdk_pb2.UpdateResponse(record=record)
```

Comment:
the `operation` field defined above doesn't seem to be used, should we yield it here, supplying the `record` created above?

Reply:
I have removed operation instance.